subgraphs/abyssmal/subgraph.yaml

```diff
@@ -13,11 +13,26 @@ dataSources:
       apiVersion: 0.0.6
       language: wasm/assemblyscript
       entities:
-        - Approval
+        - ApprovalTx
+        - ApprovalForAllTx
+        - MintCostUpdatedTx
+        - NFTMintedTx
+        - OwnershipTransferredTx
+        - TransferTx
       abis:
         - name: Abyssmal
           file: ./abis/Abyssmal.json
       eventHandlers:
         - event: Approval(indexed address,indexed address,indexed uint256)
-          handler: approval
+          handler: handleApproval
+        - event: ApprovalForAll(indexed address,indexed address,bool)
+          handler: handleApprovalsForAll
+        - event: MintCostUpdated(uint256,uint256)
+          handler: handleMintCostUpdated
+        - event: NFTMinted(indexed address,uint16,uint256)
```

Comment:
We updated the second parameter in `NFTMinted()` event to be indexed ... might want to check where else this may be relevant

`event NFTMinted(address indexed mintedBy, uint16 indexed tokenId, uint cost);`

Reply:
thanks, this was a big one